src/jit/lsraarmarch.cpp

```diff
@@ -711,6 +711,13 @@ void Lowering::TreeNodeInfoInitPutArgStk(GenTreePutArgStk* argNode, fgArgTabEntr
             MakeSrcContained(argNode, putArgChild);
         }
     }
+#ifdef _TARGET_ARM_
+    else if (putArgChild->TypeGet() == TYP_DOUBLE) {
+        // Even though TYP_DOUBLE uses 2 numSlots, we consume 1 srcCount in TYP_DOUBLE
+        argNode->gtLsraInfo.srcCount = 1;
+        putArgChild->SetContained();
```

Comment:
@BruceForstall I've been looking at the ryujit code for the first time, so I can get the wrong code. :( I faced an issue because I marked as contained so I'm trying to fix it as you said. Thank you for friendly comments!